app/packs/src/decidim/vocdoni/voter/new-vote.js

```diff
@@ -89,7 +89,7 @@ const checkIfElectionIsOpen = async (env, wallet, electionUniqueId) => {
   const client = new VocdoniSDKClient({ env, wallet })
   client.setElectionId(electionUniqueId);
   const election = await client.fetchElection();
-  const isElectionOpen = election.status === ElectionStatus.READY;
+  const isElectionOpen = election.status === ElectionStatus.ONGOING;
```

Comment:
Nice catch! There's still a missing place where to change this same constant: https://github.com/decidim-vocdoni/decidim-module-vocdoni/blob/2d347262f5a018b8a0fc4bfb2e4e6c148af48a8b/app/packs/src/decidim/vocdoni/admin/steps/vote.js#L74 
Could you handle that? Thanks

Reply:
@andreslucena, this is done

Reply:
Thanks!